File: Front/src/app/core/interfaces/producto.ts
export interface Producto {
    codigoProducto: number;
    nombre: string;
    descripcion?: string;
    referenciaInterna?: string;
    precioUnitario?: number;
    estado?: boolean;
    estadoTexto?: string;
    unidadMedida?: string;
    fechaCreacion?: Date;
}

File: Back/Productos.Common/Entities/Producto.cs
namespace Productos.Common.Entities
{
    public class Producto
    {
        public int CodigoProducto { get; set; }
        public string Nombre { get; set; } = null!;
        public string Descripcion { get; set; } = null!;
        public string ReferenciaInterna { get; set; } = null!;
        public decimal PrecioUnitario { get; set; }
        public bool Estado { get; set; }
        public string UnidadMedida { get; set; } = null!;
        public DateTime? FechaCreacion { get; set; }
    }
}
